exploratory_analysis.ipynb

```diff
@@ -1825,136 +1758,136 @@
-       "247725    KY   Madison County        21.0        151.0  Violent crime rate   \n",
-       "247718    KY    McLean County        21.0        149.0  Violent crime rate   \n",
-       "\n",
-       "        measure_id   numerator   denominator   raw_value  fips_code  \\\n",
-       "245430        43.0    9.500000  40952.000000   23.197890    18011.0   \n",
-       "247672        43.0    9.000000  13877.000000   64.855516    21135.0   \n",
-       "247732        43.0    4.666667  13304.000000   35.077170    21153.0   \n",
-       "247725        43.0  168.666667  83345.666670  202.370049    21151.0   \n",
-       "247718        43.0    2.666667   9597.333333   27.785496    21149.0   \n",
+       "                        measure_name state  county  state_code  county_code  \\\n",
+       "94               Children in poverty    TX     762         762          762   \n",
+       "142    Daily fine particulate matter    TX     762         762          762   \n",
+       "448                     Unemployment    TX     762         762          762   \n",
+       "397  Sexually transmitted infections    TX     742         742          742   \n",
+       "193               Diabetic screening    TX     741         741          741   \n",
        "\n",
-       "        start_year  \n",
-       "245430      2009.0  \n",
-       "247672      2009.0  \n",
-       "247732      2009.0  \n",
-       "247725      2009.0  \n",
-       "247718      2009.0  "
+       "     measure_id  numerator  denominator  raw_value  fips_code  start_year  \n",
+       "94          762        762            0        762        762         762  \n",
+       "142         762          0            0        762        762         762  \n",
+       "448         762        762          762        762        762         762  \n",
+       "397         742        742          731        742        742         742  \n",
+       "193         741          0          741        741        741         741  "
       ]
      },
-     "execution_count": 163,
+     "execution_count": 255,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# top five measures with the higest raw_value in group 1\n",
+    "# Top five measures with the most occurance in different states in group 1\n",
     "\n",
-    "df1.sort_values(by='measure_name', ascending=False).head()"
+    "df1_state = df1.groupby(['measure_name','state']).count().reset_index()\n",
+    "d1 = df1_state.sort_values('raw_value', ascending=False).head()\n",
+    "d1"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 164,
+   "execution_count": 258,
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "(2955, 11)"
-      ]
-     },
-     "execution_count": 164,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(9411, 11)\n",
```

Comment:
It is helpful to see the counts by measure_name, particularly for group 1, which contains year = 2010. I assume the difference in counts here is due to differences in the number of year_span values for each measure_name, given they all have the same number of state-county values. I think some more work on year_span is needed to make it useful, similar to what you did in your analysis